demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -693,6 +693,7 @@ export class DemoMeetingApp
             ) as HTMLSelectElement;
             videoInputQuality.value = '720p';
             this.audioVideo.chooseVideoInputQuality(1280, 720, 15);
+            videoInputQuality.disabled = true;
```

Comment:
For understanding: if simulcast is enabled, we want to fix the resolution to 720p for upstreaming and don't want this to be changed. Is this change for this?

Reply:
It is just demo change but its confusing if you allow them to change in the demo to 540p but the video quality in meeting is still 720p

Reply:
Cool, same as I thought, thanks for explaining!